content/docs/overview.md

```diff
@@ -1796,6 +1796,14 @@ assert(v.x == 0)
 assert(v.y == 2)
 assert(v.z == 1)
 ```
+Structs can be nested by defining a field as a struct.
```

Comment:
Or should this be more specific about struct _literals_ and say "Struct literals can be nested by defining a field as a struct literal"?